designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -120,30 +120,53 @@
 # Top-level buffers
 ###################
 
-# IOS for buffers are all false paths
+# IOs are all false paths
 set_false_path -through [get_pins ibuf_*/*]
 
+# Clock IOs should not have buffers added
+set_dont_touch_network [get_pins ibuf_*/in*]
+set_dont_touch_network [get_pins ibuf_*/clk*]
+
 #############
 # Analog core
 #############
 
-# TODO do any signals in the debug interface need special treatment?
+# Debugging signals are all false paths
 set_false_path -through [get_pins iacore/adbg_intf_i_*]
 
+# Clock outputs in the debug interface should not have buffers added
+set adbg_clk_pins [get_pins {{ \\
```

Comment:
looks OK -- this used to be called `mon_nets` and is just being renamed `adbg_clk_pins`